aten/src/ATen/native/native_functions.yaml

```diff
@@ -8537,6 +8537,11 @@
     CPU: _convert_indices_from_csr_to_coo_structured_cpu
     CUDA: _convert_indices_from_csr_to_coo_structured_cuda
 
+- func: _csr_to_block_csr(Tensor self, int[2] block_size) -> Tensor
```

Comment:
Even though it's prefixed with an underscore for now, let's maybe put it inside the `torch.sparse` module?
`python_module: sparse` would need to add here.
It would be available in Python as `torch._C._sparse._csr_to_block_csr` which is usually bound inside `torch/sparse/__init__.py` to `torch.sparse` module.

Reply:
Sure fine by me :)